docs/architecture/smart-rollups.md

```diff
@@ -71,6 +71,26 @@ Each block also contains the following internal messages, which are created by t
 
 Smart Rollup nodes can use these internal messages to know when blocks begin and end.
 
+## Commitments
+
+Some Smart Rollup nodes post commitments to layer 1, which include a hash of the current state of the kernel.
+If any node's commitment is different from the others, the protocol compares them and identifies the correct commitment.
+This process ensures the security of the Smart Rollup by verifying that the nodes are running the kernel faithfully.
+
+Only Smart Rollup nodes running in operator or maintenance mode post these commitments on a regular basis.
+Nodes running in other modes such as observer mode run the kernel and monitor the state of the Smart Rollup just like nodes in operator or maintenance mode, but they do not post commitments.
+
+If nodes post commitments that don't match, they play a refutation game to determine the correct commitment, eliminate incorrect commitments, and penalize the nodes that posted incorrect commitments.
+
+## Bonds
+
+When a user runs a node that posts commitments, the protocol automatically locks a bond of 10,000 liquid, unstaked tez from user's account as assurance that they are running the kernel faithfully.
```

Comment:
I might be wrong but that 10k can be staked. It's a long time I haven't check that code.

Reply:
From internal discussion, it seems that the tez must be liquid and unstaked.